bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java

```diff
@@ -337,20 +338,27 @@ public Collection<WorkItem> prRun(Path scratchPath) {
                 var text = "This backport pull request has now been updated with the original issue," +
                         " but not the original commit. If you have the original commit hash, please update" +
                         " the pull request title with `Backport <hash>`.";
-                pr.addComment(text);
+                var comment = pr.addComment(text);
                 pr.addLabel("backport");
-                return List.of(new CheckWorkItem(bot, prId, errorHandler));
+                logLatency("Time from PR updated to backport comment posted ", comment.createdAt(), log);
+                return List.of(new CheckWorkItem(bot, prId, errorHandler, prUpdatedAt));
             }
 
             // If the title needs updating, we run the check again
             if (updateTitle()) {
-                return List.of(new CheckWorkItem(bot, prId, errorHandler));
+                var updatedPr = bot.repo().pullRequest(prId);
+                logLatency("Time from PR updated to title corrected ", updatedPr.updatedAt(), log);
+                return List.of(new CheckWorkItem(bot, prId, errorHandler, prUpdatedAt));
             }
 
             try {
                 Repository localRepo = materializeLocalRepo(scratchPath, hostedRepositoryPool);
 
                 var expiresAt = CheckRun.execute(this, pr, localRepo, comments, allReviews, activeReviews, labels, census, bot.ignoreStaleReviews(), bot.integrators());
+                if (log.isLoggable(Level.INFO)) {
+                    var updatedPr = bot.repo().pullRequest(prId);
+                    logLatency("Time from PR updated to CheckRun done ", updatedPr.updatedAt(), log);
```

Comment:
logLatency only logs at INFO level anyway. I'm not sure why you put just this one instance inside the log.isLoggable guard.

Reply:
Actually, I'm trying to understand why this is correct. `logLatency` looks like this:
```
    protected void logLatency(String message, ZonedDateTime endTime, Logger log) {
        var latency = Duration.between(prUpdatedAt, endTime);
...
```

I would have expected the endTime to be the current time. But maybe I'm just getting lost here on the difference between `updatedPr.updatedAt()` and `prUpdatedAt`.

Reply:
We basically always log INFO so the guard isn't strictly necessary. Even so I tried to put guards whenever I needed to fetch data from a remote to generate the log message. In this case, it's refetching the PullRequest from the forge.

I should add a comment about this to make it clearer. Whenever possible, I want the latency numbers to be calculated between timestamps from the same clock. In this case the clock is on the forge (e.g. GitHub). The start time is the updatedAt timestamp read from the PR when we triggered this WorkItem. The endTime is the updatedAt from the PR just now, after we updated it from this WorkItem (CheckRun.execute() will always update the `check` in the PR).